docs/metal.md

```diff
@@ -38,6 +38,7 @@ Command line interface for Equinix Metal
 * [metal init](metal_init.md)	 - Create a configuration file.
 * [metal interconnections](metal_interconnections.md)	 - interconnections operations: create, get, update, delete
 * [metal ip](metal_ip.md)	 - IP address, reservations, and assignment operations: assign, unassign, remove, available, request and get.
+* [metal loadbalancer-beta](metal_loadbalancer-beta.md)	 - LoadBalancer BETA operations: create, get, update, delete, and bgpconfig.
```

Comment:
The `metal_loadbalancer-beta.md` doesn't appear to exist. And I think we can drop the `bgpconfig` bit from this line

Reply:
🤦 `git add` ... 
`bgpconfig` is a copy/paste from using the Project subcommand as the base. I tried to catch these but looks like I missed some.